config/initializers/fog_setting.rb

```diff
@@ -1,11 +1,19 @@
  CarrierWave.configure do |config|
-     config.fog_credentials = {
+    # if Rails.env.development?
+    #   config.storage = :file
+    #   config.enable_processing = false
+    # else
+    # config.storage = :fog
+    config.fog_provider = 'fog/aws'
+    config.fog_credentials = {
        provider:              'AWS',
        aws_access_key_id:     ENV['AWS_ACCESS_KEY_ID'] || '',
        aws_secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'] || '',
        region:                'us-west-2'
      }
+     # Change folder name to s3 bucket name in production env
      config.fog_directory  = 'awesomesanswer22e'
```

Comment:
We could save the BUCKET_NAME in environment variable, but I think it's not a must. People cannot use the bucket if they don't have the access key and secret key.